csf_tz/hooks.py

```diff
@@ -10,7 +10,7 @@
 app_color = "green"
 app_email = "info@aakvatech.com"
 app_license = "GNU General Public License (v3)"
-
+required_apps = ["frappe/erpnext", "frappe/hrms"]
```

Comment:
Is this the correct syntax? Isn't it supposed to be ["frappe", "erpnext", "hrms"]